docker-entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/dumb-init /bin/sh
```

Comment:
What is `/bin/dumb-init` ?

Reply:
https://github.com/Yelp/dumb-init - the `runatlantis/atlantis` Docker  image uses it. I found this interesting https://github.com/Yelp/dumb-init#session-behavior